packages/eslint-plugin-ckeditor5-rules/lib/rules/no-relative-imports.js

```diff
@@ -15,31 +20,97 @@ module.exports = {
 			url: 'https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/code-style.html#importing-between-packages-ckeditor5-rulesno-relative-imports'
 		},
 		fixable: 'code',
-		schema: []
+		schema: [
+			{
+				type: 'object',
+				properties: {
+					newInstallationMethod: {
+						type: 'boolean'
+					}
+				},
+				additionalProperties: false
+			}
+		]
 	},
 	create( context ) {
+		const [ { newInstallationMethod } = {} ] = context.options;
+
 		return {
-			ImportDeclaration: node => {
-				const importPath = node.source.value;
-				const indexOfLastDoubleDots = importPath.indexOf( '../ckeditor5-' );
-
-				if ( indexOfLastDoubleDots >= 0 ) {
-					context.report( {
-						node,
-						message: 'Imports of CKEditor5 packages shouldn\'t be relative.',
-						fix: fixer => {
-							// The range starts after the quote sign.
-							const rangeStart = node.source.range[ 0 ] + 1;
-
-							// The range end is the length of '../' string plus the index of '../ckeditor5-' in path.
-							const rangeEnd = rangeStart + indexOfLastDoubleDots + 3;
-
-							// Replace relative path in import '../[../]{0,n}' with '@ckeditor/' to make package import.
-							return fixer.replaceTextRange( [ rangeStart, rangeEnd ], '@ckeditor/' );
-						}
-					} );
-				}
-			}
+			ImportDeclaration: newInstallationMethod ? getNIMRule( context ) : getLegacyRule( context )
 		};
 	}
 };
+
+function getLegacyRule( context ) {
+	return node => {
+		const importPath = node.source.value;
+		const indexOfLastDoubleDots = importPath.indexOf( '../ckeditor5-' );
+
+		if ( indexOfLastDoubleDots >= 0 ) {
+			context.report( {
+				node,
+				message: MESSAGES.relative,
+				fix: fixer => {
+					// The range starts after the quote sign.
+					const rangeStart = node.source.range[ 0 ] + 1;
+
+					// The range end is the length of '../' string plus the index of '../ckeditor5-' in path.
+					const rangeEnd = rangeStart + indexOfLastDoubleDots + 3;
+
+					// Replace relative path in import '../[../]{0,n}' with '@ckeditor/' to make package import.
+					return fixer.replaceTextRange( [ rangeStart, rangeEnd ], '@ckeditor/' );
+				}
+			} );
+		}
+	};
+}
+
+function getNIMRule( context ) {
+	return node => {
+		const importPath = node.source.value;
+
+		// CSS imports.
+		if ( !node.specifiers.length ) {
+			return;
+		}
+
+		const pathMatch = importPath.match( /([^']+)\/(ckeditor5-[^/]+)([^']+)?/ );
+
+		if ( !pathMatch ) {
+			return;
+		}
+
+		const [ , prefix, packageName, suffix ] = pathMatch;
+		const usesRelativeImport = prefix !== '@ckeditor';
```

Comment:
It looks like this is too greedy, because it targets not only relative paths like `../ckeditor5-foo/path/to/file.js` but also `@another-pkg/ckeditor5-foo/path/to/file.js`, which is not a relative path.